README.md

```diff
@@ -46,6 +46,19 @@ plugins:
    - '@silvermine/serverless-plugin-cloudfront-distribution-tags'
 ```
 
+Since CloudFront distributions can take quite a while to deploy, you likely only want to
+have to CloudFormation update them when changes are actually made. Unfortunately, when the
+tags on a distribution change, CloudFormation will redeploy the distribution. If you have
+a tag who's value changes even when the distribution may not have, e.g. `CODE_VERSION`,
```

Comment:
This would be "whose" (https://www.grammarly.com/blog/whos-whose/), but maybe it's clearer to avoid personifying a tag at all. Perhaps "If you have a tag with a value that changes..."